<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab5ChartPlot</class>
 <widget class="QWidget" name="Tab5ChartPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,2,2,2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pLExAxis"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pLEyAxis"/>
     </item>
     <item>
      <widget class="QPushButton" name="pPBaddButton">
       <property name="text">
        <string>add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBclearButton">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pChartHoriLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
